environments/Test-Laptop.json

```diff
@@ -41,6 +42,28 @@
         "hdfs" : {
           "dfs_replication_factor" : 1
         },
+        "realm" : "BCPC.EXAMPLE.COM",
+        "enable_kerberos": false,
+        "kerberos": {
+          "data": {
```

Comment:
Is the `keytab` key in these hashes effectively the file name since the hash data structure is really always keyed off the key like `namenode`, `datanode`, etc.? If so, couldn't the `keytab` entry be omitted an generated to make this data structure more simple?